### PR TITLE
Main

### PR DESCRIPTION
**Pull request by manan16**

### CODE REVIEW

**Review by manan16 — COMMENTED** [review]
just in case user wants to input how many values he requires....

File: main.cpp [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
 
-int main (){
+int main ()
+{
     Fibonancci fib; 
-    int total=0;
-    for (int i = 0; i < 20; i++)
-        total += fib.produce_output(i);
-    cout << total << endl;
+    int term=0,n;
```
Comment (RyanGouldsmith) [inline main.cpp]: Nice one for raising a PR! Could we name `n` something like `userInput` 😄

Comment (manan16) [reply main.cpp]: it depends upon u

Comment (manan16) [reply main.cpp]: :)

**Pull request merged**